src/styles/technology.scss

```
@@ -0,0 +1,132 @@
+.technology {
+  &__photo {
+    width: 100%;
+    height: 388px;
+    overflow: hidden;
+  }
+
+  &__photo--1 {
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+
+    &:hover {
+      transform: scale(1.05);
+  
+      transition: transform 0.3s;
+    }
+  }
+
+  &__photo--2 {
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+
+    &:hover {
+      transform: scale(1.05);
+  
+      transition: transform 0.3s;
+    }
+  }
+
+  &__description {
+    margin-top: 30px;
+    color: #7c7c7c;
+
+    @include onMobile {
+      margin-top: 20px;
+    }
+  }
+
+  &__link {
+    margin-top: 60px;
+
+    position: relative;
+    width: fit-content;
+
+    @include onMobile {
+      margin-top: 30px;
+    }
+  }
+
+  &__link-set {
+    font-size: 12px;
+    line-height: 15px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $c-green;
+
+    &::after {
+      content: "";
+      position: absolute;
+      bottom: -6px;
+      width: 54px;
+      left: 0;
+      height: 1px;
+      background-color: $c-green;
+      transition: 0.3s;
```

don't use it like that, because it's the same as transition: all 0.3s or transition-durations: 0.3s
It's too hard for the browser
also, create variable for transition duration
```suggestion
      transition: someValue $transitionDuration;
```